### DISCUSSION TITLE
Please tell me how to add a custom icon for the health icon in KADE Engine 1.7.

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1993] [Q&A]
Please tell me how to add a custom icon for the health icon in KADE Engine 1.7.

**Comment by SuxxorState** [comment]
if you've already created a custom character, just toss in the two icons into a 300×150 png (normal then defeat icon), name this icon-yourcharacter.png (yourcharacter should be the name in the debug menu) and put the file in the icons folder in images
